base/src/main/java/com/smartdevicelink/managers/StreamingStateMachine.java

```diff
@@ -37,12 +37,12 @@
 import java.lang.annotation.RetentionPolicy;
 
 public class StreamingStateMachine {
-    @IntDef({NONE, READY, STARTED, STOPPED, ERROR})
+    @IntDef({NONE, READY, STARTED, STOPPED, ERROR, PAUSED})
```

Comment:
To maintain alignment with the iOS library and since the proposal is to Stop the Video stream and then start it again I recommend that we should not add a PAUSED state and should use the STOPPED state where appropriate

Reply:
@RHenigan this was requested by @joeljfischer previously. Are you sure we should remove this state?

Reply:
@kostyaBoss although this matter was discussed in the review it is not reflected in the final proposal and since the VideoStreamService should be stopped and then started again in this use case I believe that STOPPED state should be used.